### PR TITLE
VPN-4923: Accessibility Principles and Requirements doc

### PR DESCRIPTION
**Pull request by vinocher**
## Description

  Document that outlines Accessibility Principles and Requirements for VPN Client

## Reference

  [VPN-4923](https://mozilla-hub.atlassian.net/browse/VPN-4923), [Github issue 7090](https://github.com/mozilla-mobile/mozilla-vpn-client/issues/7090)

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

[VPN-4923]: https://mozilla-hub.atlassian.net/browse/VPN-4923?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/Accessibility - Principles and Requirements.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+     - All functionality and interactive elements must be accessible solely
+      using a keyboard, without relying on mouse or touch interactions.
+
+4. **Text Contrast Compliance:**
+
+     - The default text contrast against its background must meet the
+      standards set by
+      [WCAG](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast).
+
+     - Users with vision impairments should be able to apply the
```
Comment (lesleyjanenorton) [inline docs/Accessibility - Principles and Requirements.md:79]: Do we think `High Contrast Theme Support` should be it's own numbered item?

File: docs/Accessibility - Principles and Requirements.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+5. **Screen Magnification:**
+
+     - All screens must be readable when using the screen magnification
+      tool.
+
+     - Any focused element should automatically scroll into the
+      magnification tool\'s viewport.
+
+### Lower Priority Accessibility Requirements
```
Comment (lesleyjanenorton) [inline docs/Accessibility - Principles and Requirements.md:91]: Nit: Lower Priority Accessibility *Considerations*, maybe?  I'm probably being too pedantic but "Lower Priority" seems a bit at odds with "Requirement".

File: docs/Accessibility - Principles and Requirements.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      Client should be compatible with the major ATs of the platforms
+      that it runs on, including Text Size, Screen Readers, Keyboard
+      Access, Text Contrast, Screen Magnification, and others.
+
+3. **Alignment with Established Standards:** The VPN Client should try
+      to meet accessibility standards like
+      [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
+      where applicable, even though the VPN Client is not a Web App.
+
+4. **Foster Collaborative Development and Testing:** Promote a
```
Comment (lesleyjanenorton) [inline docs/Accessibility - Principles and Requirements.md:37]: ❤️

File: docs/Accessibility - Principles and Requirements.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+Accessibility is an important feature that ensures that all users,
+including those with disabilities, can use the Mozilla VPN Client
+effectively. Following accessibility principles and requirements
+enhances the user experience, reaches a broader audience, and complies with
+accessibility standards across all supported platforms.
+
+## Principles
+
+1. **Inclusive design:** The VPN Client should be designed for use by
```
Comment (lesleyjanenorton) [inline docs/Accessibility - Principles and Requirements.md:21]: ❤️

**Review by lesleyjanenorton — APPROVED** [review]
Thank you @vinocher !

File: docs/Accessibility - Principles and Requirements.md (hunk 2 above)
Comment (vinocher) [reply docs/Accessibility - Principles and Requirements.md:91]: Sounds good.

File: docs/Accessibility - Principles and Requirements.md (hunk 1 above)
Comment (vinocher) [reply docs/Accessibility - Principles and Requirements.md:79]: Yes, will do.

**Pull request merged**